types/hapi/test/route/route-options.ts

```diff
@@ -103,7 +103,7 @@ const routeOptions: RouteOptions = {
     auth: routeOptionsAccess,
     bind: null,
     cache: {
-        privacy: 'default',
+        privacy: RouteOptionsPrivacy.Default,
```

Comment:
Actually, this brings up one question, if we enable `preserveConstEnums`, will this break?
hapi doesn't actually export any enums, it might actually be a bad idea to use string enums and instead use type unions again.

Reply:
Good point, commit incoming with all enums removed